src/curlpp/Easy.cpp

```diff
@@ -64,10 +64,10 @@ curlpp::Easy::setOpt(const OptionBase & option)
 
 
 void
-curlpp::Easy::setOpt(std::auto_ptr<OptionBase> option)
+curlpp::Easy::setOpt(std::shared_ptr<OptionBase> option)
 {
 	option->updateHandleToMe(mCurl.get());
-	mOptions.setOpt(option.release());    
+	mOptions.setOpt(option.get());    
```

Comment:
Ok, I'll test the lib again using unique_ptr. When I get this tested I'll request pull again!